### ISSUE TITLE
Wrong  scope for dependencies in maven pom file  io.arrow-kt:arrow-fx-kotlinx-coroutines:0.10.5

### ISSUE THREAD

**Issue by routis** [arrow-kt/arrow-fx#139]
The newly released io.arrow-kt:arrow-fx-kotlinx-coroutines:0.10.5 doesn't seem to have a correct pom file.
It declares dependencies to three libraries:
- io.arrow-kt:arrow-fx:0.10.5
- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.61
- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3
Unfortunately those 3 dependencies are declared in scope "runtime" which should have been "compile". So, this means that a project that depends on arrow-fx-kotlinx-coroutines needs to re-declare those dependencies in the correct scope.

**Comment by rachelcarmena** [comment]
Thanks @routis for raising this issue :raised_hands: 

They aren't declared in the right scope:

```
<dependency>
<groupId>io.arrow-kt</groupId>
<artifactId>arrow-fx</artifactId>
<version>0.10.5</version>
<scope>runtime</scope>
</dependency>
<dependency>
<groupId>org.jetbrains.kotlin</groupId>
<artifactId>kotlin-stdlib-jdk7</artifactId>
<version>1.3.61</version>
<scope>runtime</scope>
</dependency>
<dependency>
<groupId>org.jetbrains.kotlinx</groupId>
<artifactId>kotlinx-coroutines-core</artifactId>
<version>1.3.3</version>
<scope>runtime</scope>
</dependency>
```

**Comment by rachelcarmena** [comment]
It's already fixed for `0.11.0-SNAPSHOT`. View last POM file in OSS repository:

https://oss.jfrog.org/artifactory/oss-snapshot-local/io/arrow-kt/arrow-fx-kotlinx-coroutines/0.11.0-SNAPSHOT/

Thanks again @routis :raised_hands:
